pkg/api/handlers/libpod/networks.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/containers/podman/v4/pkg/api/handlers/utils"
 	api "github.com/containers/podman/v4/pkg/api/types"
 	"github.com/containers/podman/v4/pkg/domain/entities"
+	"github.com/containers/podman/v4/pkg/domain/entities/reports"
```

Comment:
I had no idea this package existed but question why?  I prefer to keep things consistent and have all `*Report` structures in `entities` and remove `entities/reports`.

@containers/podman-maintainers PTAL